gordo_components/builder/build_model.py

```diff
@@ -125,6 +129,28 @@ def build_model(
     return model, metadata
 
 
+def _determine_offset(model: BaseEstimator, X: Union[np.ndarray, pd.DataFrame]) -> int:
```

Comment:
Nice, it did not even cross my mind that we could just do this and calculate the offset instead of having the model tell us about it!